odoo/addons/test_assetsbundle/tests/test_assetsbundle.py

```diff
@@ -80,33 +80,23 @@ def test_operations(self):
 
 
 class AddonManifestPatched(TransactionCase):
-
-    test_assetsbundle_manifest = None
-    for path in ADDONS_PATH:
-        manifest = read_manifest(path, 'test_assetsbundle')
-        if manifest:
-            manifest['addons_path'] = path
-            test_assetsbundle_manifest = manifest
-            break
-
-    def tearDown(self):
-        super().tearDown()
-        self.env.registry._init_modules = self.__genuine_registry_modules
-        http.addons_manifest = self.__genuine_addons_manifest
-
     def setUp(self):
         super().setUp()
-        self.__genuine_registry_modules = self.env.registry._init_modules
-        self.env.registry._init_modules = func.lazy(lambda: set(self.installed_modules))
 
-        self.__genuine_addons_manifest = http.addons_manifest
-        http.addons_manifest = func.lazy(lambda: self.manifests)
-
-        self.installed_modules = ['base', 'test_assetsbundle']
+        self.installed_modules = {'base', 'test_assetsbundle'}
         self.manifests = {
-            'test_assetsbundle': self.test_assetsbundle_manifest,
+            'test_assetsbundle': load_manifest('test_assetsbundle'),
         }
 
+        self.patch(self.env.registry, '_init_modules', self.installed_modules)
+        self.patch(load_manifest.__wrapped__, 'func', Mock(side_effect=lambda module: self.manifests.get(module, {})))
+        load_manifest.cache_clear()
+
+    @classmethod
+    def tearDownClass(cls):
+        super().tearDownClass()
+        load_manifest.cache_clear()
```

Comment:
```suggestion
    self.addCleanup(load_manifest.cache_clear)
```

Reply:
removed as mockable is now before lur_cache